tests/pytests/unit/conftest.py

```diff
@@ -1,8 +1,25 @@
+import os
+
 import pytest
 
 import salt.config
 
 
+@pytest.fixture(scope="package", autouse=True)
+def onedir_env():
```

Comment:
```suggestion
def _onedir_env():
```
Since it's an auto use fixture, let's prefix it with `_` so that it gets hidden on the possible fixtures to use shown by pytest.